videoprocessor/resources/videoSplitter.py

```diff
@@ -27,7 +31,10 @@ def videosplitter(key):
 
             # Publishing frames to kafka topic
             kafka_producer = connect_kafka_producer()
-            publish_message(kafka_producer, config.Config.KAFKA_TOPIC, 'frame', key, imageData)
+            if saga_enabled is None:
+                saga_enabled = config.Config.SAGA_ENABLED
+            if str(saga_enabled) == '1':
+                publish_message(kafka_producer, config.Config.KAFKA_TOPIC, 'frame', key, imageData)
```

Comment:
This change should be reverted, is not required.

Reply:
its already reverted

Reply:
done